build.sbt

```diff
@@ -1,13 +1,27 @@
 lazy val root = (project in file("."))
   .settings(basicSettings: _*)
+  .settings(
+    libraryDependencies ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n <= 12 =>
+	  List(compilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full))
+	case _ => Nil
+      }
+    },
+    Compile / scalacOptions ++= {
+      CrossVersion.partialVersion(scalaVersion.value) match {
+        case Some((2, n)) if n <= 12 => Nil
+	case _                       => List("-Ymacro-annotations")
+      }
+    }
+  )
   .settings(dependencySettings: _*)
-  .settings(addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full))
 
 lazy val basicSettings = Seq(
   name := "config-annotation",
   organization := "com.wacai",
-  scalaVersion := "2.11.5",
-  crossScalaVersions := Seq(scalaVersion.value, "2.12.0"),
+  scalaVersion := "2.13.4",
+  crossScalaVersions := Seq(scalaVersion.value, "2.12.13", "2.11.12"),
```

Comment:
Please let scala versions in `.travis.yml` be the same with `crossScalaVersions`.